react/ProductSKUNames.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { useCssHandles } from 'vtex.css-handles'
+import useProduct from 'vtex.product-context/useProduct'
+
+const CSS_HANDLES = [
+  'productSKUNamesContainer',
+  'skuLineClass',
+  'skuNameClass',
+  'skuValueClass',
```

Comment:
I think the name of this file and their CSS handles should be related to `ProductSKUAttributes` instead os skunames